server/src/core/chat.ts

```
@@ -27,18 +37,52 @@ function attemptToSendChatMessage(
+      if (roomIndex === -1) {
+        log.warn(
+          `Room doesn't exist for Socket ID ${connectionID} (${playerName}) when validating chat message.`
+        );
+        return false;
+      }
       if (!validateMessage(message, connectionID)) {
         log.warn(`Bad chat validation for ${connectionID} (${playerName})`);
         return;
       }
-
       let room = findRoomWithConnectionID(connectionID, true) as Room;
```

Biome

[error] This let declares a variable that is only assigned once.

'room' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)